e2e/ansible/templates/hyperconverged/ubuntu/Vagrantfile.hyperconverged

```diff
@@ -12,7 +12,7 @@ Vagrant.configure("2") do |config|
 
   # Every Vagrant development environment requires a box. You can search for
   # boxes at https://atlas.hashicorp.com/search.
-  config.vm.box = "ubuntu/xenial64"
+config.vm.box = "openebs/k8s-1.8.8-ubuntu"
```

Comment:
This change is not required, it is done by setup-env.yml

Reply:
Ok @yudaykiran I will revert. This change was done by setup-env.yml

Reply:
Does it go and modify the file in /ubuntu/Vagrantfile.hyperconverged or the file copied from that location to playbook_dir?

Reply:
It will modify the Vagrantfile in playbook_dir location.

Reply:
Done. Reverted the changes